datasets/gov/finmin/paraiskos.csv

```diff
@@ -0,0 +1,91 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/finmin/paraiskos,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,Project,,,project_id,,,4,,,Projektas ir jo vertinimai,
+,,,,,project_id,string,,,,4,open,,Projekto identifikatorius,
+,,,,,call,ref,/datasets/gov/finmin/kvietimai/Call,,,4,open,,Projekto kvietimo/patvirtinto sąrašo identifikatorius,
+,,,,,code,string,,,,4,open,,Projekto kodas,
+,,,,,state,string,,,,3,open,,Projekto būsena,
+,,,,,state_date,date,D,,,4,open,,Projekto būsenos įgijimo data,
+,,,,,status,string,,,,3,open,,Projekto etapas,
+,,,,,eligibility_result,string,,,,3,open,,Tinkamumo vertinimo rezultatas,
+,,,,,eligibility_result_date,date,D,,,4,open,,Tinkamumo vertinimo rezultato data,
+,,,,,eligibility_request_amount,number,,,,2,open,,Tinkamumo vertinimo prašomų lėšų suma,
+,,,,,,comment,type,,"update(type: ""money"", ref: ""EUR"")",4,,spinta:40,,
+,,,,,eligibility_expense_amount,number,,,,2,open,,Tinkamumo vertinimo netinkamų išlaidų suma,
+,,,,,,comment,type,,"update(type: ""money"", ref: ""EUR"")",4,,spinta:40,,
+,,,,,benefit_result,string,,,,3,open,,Naudos ir kokybės vertinimo rezultatas,
+,,,,,benefit_result_date,date,D,,,4,open,,Naudos ir kokybės vertinimo rezultato data,
+,,,,,benefit_score,number,,,,3,open,,Vertinimo metu suteiktų balų skaičius,
```

Comment:
Kodėl nurodote 3 brandos lygį?

Reply:
Balai nurodyti be jokios skalės, ir vienetai tokie nekonkretūs - nežinau, koks lygis tinkamiausias

Reply:
Ačiū už paaiškinimą. Galim palikti 3 brandos lygį.